WooCommerce/Classes/ViewRelated/Point Of Sale/PointOfSaleDashboard.swift

```diff
@@ -0,0 +1,8 @@
+import SwiftUI
+
+
+struct PointOfSaleDashboard: View {
+    var body: some View {
+        Text("WooCommerce Point Of Sale")
```

Comment:
I don't think we need to encapsulate this into a view model, or separately for the time being, since is just dummy text. Happy to iterate thought!